libchisel/src/lib.rs

```diff
@@ -45,6 +45,12 @@ pub trait ModulePreset {
         Self: std::marker::Sized;
 }
 
+impl From<String> for ModuleError {
```

Comment:
Could perhaps also implement `From<&'a str>` as well.

Reply:
Do you want to implement that in a new PR? I'm for just merging this.

Reply:
ok, sounds good